src/config.rs

```diff
@@ -23,32 +22,63 @@ const DEFAULT_SYMLINK_ROOT: &'static str = "/var/run/gpio";
 #[derive(Debug, PartialEq, Clone)]
 pub struct Direction(pub sysfs_gpio::Direction);
 
+#[derive(Deserialize, Debug)]
+#[serde(remote = "sysfs_gpio::Direction")]
+pub enum DirectionDef {
```

Comment:
I believe you can probably just do `#[serde(rename_all = "lowercase")]`

Reply:
I believe `Direction` is coming out of `sysfs_gpio`, so we'll have to edit it there.